sample-app/index.html

```diff
@@ -0,0 +1,286 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
could you add a screenshot or video of the web running?

Reply:
<img width="1727" alt="Screenshot 2024-07-25 at 4 40 21 AM" src="https://github.com/user-attachments/assets/a602a248-fe4b-449e-a257-ba38c4e5e59e">
<img width="1727" alt="Screenshot 2024-07-25 at 4 45 21 AM" src="https://github.com/user-attachments/assets/7412fdb3-d019-4249-8800-f5605ba76dad">